sdk/storage/azure-storage-blob/HISTORY.md

```diff
@@ -6,7 +6,8 @@
 
 - `set_container_access_policy` has required parameter `signed_identifiers`.
 - NoRetry policy has been removed. Use keyword argument `retry_total=0` for no retries. 
-
+- `AccountSasPermissions`, `BlobSasPermissions`, `ContainerSasPermissions`, `ResourceTypes`, and `Services` 
```

Comment:
The first three ones are already done in preview 4..You want to remove them from this.